packages/edit-site/src/components/routes/redirect-to-homepage.js

```diff
@@ -48,7 +55,7 @@ async function getHomepageParams( siteUrl ) {
 		.then( ( { data } ) => data );
 
 	if ( ! template?.id ) {
-		return;
+		throw new Error( '`getHomepageParams`: unable to find home template.' );
```

Comment:
**suggestion:** Maybe we should make this message more user-friendly and display it instead of a generic error message?

```
The editor is unable to find a block template for the home page.
```

What do you think?

Reply:
Yep, sounds good 👍

Reply:
So I've kept these error messages the same so that we can locate them in the code when the user uses 'Copy Error', but the visible message is now what you suggested.